lib/Dancer2/Core/Role/Config.pm

```diff
@@ -281,8 +255,46 @@ __END__
 
 =head1 DESCRIPTION
 
-Provides a C<config> attribute that feeds itself by finding and parsing
-configuration files.
+This is the redesigned C<Dancer2::Core::Role::ConfigReader>
+to manage the Dancer2 configuration.
+
+It is now possible for user to control which B<ConfigReader>
+class to use to create the config.
```

Comment:
"Users can control which B<ConfigReader> class to use when creating the config."

Same comment as above.